<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>413</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1313</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonU">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icons/icons8-u-50.png</normaloff>:/images/icons/icons8-u-50.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonI">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icons/icons8-i-50.png</normaloff>:/images/icons/icons8-i-50.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icons/icons8-b-50.png</normaloff>:/images/icons/icons8-b-50.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="colorButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/icons/icons8-swatch-50.png</normaloff>:/images/icons/icons8-swatch-50.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxFontSize">
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="labelIsSaved">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/icons/icons8-delete-document-50.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="charCounter">
      <property name="text">
       <string>Characters: 0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="wordCounter">
      <property name="text">
       <string>Words: 0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_document"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRedo"/>
    <addaction name="actionEdit"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_document"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_document">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-alpaca-50.png</normaloff>:/images/icons/icons8-alpaca-50.png</iconset>
   </property>
   <property name="text">
    <string>New document</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-cat-50.png</normaloff>:/images/icons/icons8-cat-50.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-chimpanzee-50.png</normaloff>:/images/icons/icons8-chimpanzee-50.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-dog-50.png</normaloff>:/images/icons/icons8-dog-50.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-orca-50.png</normaloff>:/images/icons/icons8-orca-50.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-pig-50.png</normaloff>:/images/icons/icons8-pig-50.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-turtle-50.png</normaloff>:/images/icons/icons8-turtle-50.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-unicorn-50.png</normaloff>:/images/icons/icons8-unicorn-50.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-shark-50.png</normaloff>:/images/icons/icons8-shark-50.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/icons8-bug-50.png</normaloff>:/images/icons/icons8-bug-50.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fontComboBox</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>textEdit</receiver>
   <slot>setCurrentFont(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontWeight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonU</sender>
   <signal>clicked(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonI</sender>
   <signal>clicked(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
